applications/LinearSolversApplication/custom_solvers/eigen_pardiso_llt_solver.h

```diff
@@ -43,10 +43,32 @@ class EigenPardisoLLTSolver
 
     void Initialize(Parameters settings)
     {
+        // Set default parameters
+        if (!settings.Has("num_threads_mkl")) {
+            settings.AddEmptyValue("num_threads_mkl").SetString("consistent"); // Default to 0 (consistent)
```

Comment:
```suggestion
            return;
```

this shouldnt do anything if it isnt defined